src/user/dto/create-user.dto.ts

```
@@ -0,0 +1,14 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { IsNotEmpty, IsOptional } from "class-validator";
+
+export class CreateUserDto {
+
+    @ApiProperty({ example: 'This is username' })
```

ESLint

[error] Delete `⏎··`

(prettier/prettier)